vite.config.ts

```diff
@@ -1,7 +1,15 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+/// <reference types="vitest" />
+/// <reference types="vite/client" />
```

Comment:
commented code?

Reply:
This is actually not a commented code, it is required to include TypeScript reference directives for using vitest.